source/extensions/transport_sockets/tls/BUILD

```diff
@@ -30,8 +30,14 @@ envoy_cc_extension(
 
 envoy_cc_library(
     name = "ssl_socket_lib",
-    srcs = ["ssl_socket.cc"],
-    hdrs = ["ssl_socket.h"],
+    srcs = [
```

Comment:
Can io_handle_bio.{cc,h} in a separate envoy_cc_library?

Reply:
Done!